frontends/web/src/routes/account/send/utxos.jsx

```diff
@@ -71,7 +71,6 @@ export default class UTXOs extends Component {
         t,
         active,
     }, {
-        show,
```

Comment:
Remove it from the whole component.

show() and hide() also not used.

Also seems like a new bug: when coin control was hidden via `hide()`, `this.props.onChange(this.state.selectedUTXOs);` was called, so the send.jsx parent component erased the selected utxos. Probably not happening anymore. 

Pls fix all of that at once, or make an issue to do it later :)